index.html

```diff
@@ -61,27 +61,323 @@
       </header>
 
       <main>
-        <div class="auth__container absolute-container">
-          <h2 class="heading">Login</h2>
-          <form>
-            <div class="text-field">
-              <input type="email" required class="form-input" />
-              <span></span>
-              <label>Email Address</label>
+        <div class="hero__container">
+          <button class="hero-btn primary-btn">Shop Now</button>
+        </div>
+        <div class="hero-heading">Featured Categories</div>
+        <div class="hero__categories">
+          <div class="item item-1">
+            <p class="item-name">DOG FOOD</p>
+          </div>
+          <div class="item item-2">
+            <p class="item-name">TOYS</p>
+          </div>
+          <div class="item item-3">
+            <p class="item-name">CAT FOOD</p>
+          </div>
+          <div class="item item-4">
+            <p class="item-name">TREATS</p>
+          </div>
```

Comment:
UI improvement- change the text color for featured categories to make it readable

Reply:
sure will add some bright colour. Thank you.